### ISSUE TITLE
No speech feedback using Ctrl F in 4.7.2 beta for range selection

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#33137]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When a bar number or range of bar numbers is entered using Ctrl F the screen reader gives no verbal feedback when the command is executed and jump/selection made.  This has confused blind users for some time as they don't realise they have jumped and press Esc to close the pop up (which is already closed) and end up back at the start of the score.  But with the range selection this is particularly difficult as you can't use arrow movement to confirm where you are before doing whatever you wanted to make the selection for, e.g. deleting, adding hairpin, articulation, etc.

Being able to select in this way is an exciting development for blind users as selecting a range is quite difficult, with keyboard, especially if it is long or covers multiple lines, sometimes not even possible.  But not knowing the selection has been made or what has been selected might cause frustration or even errors.

STEPS:
1.  On a score with at least 20 bars press Ctrl F, type 12 and press enter.  Notice there is no speech feedback of any sort to know it has moved.
2. Press Esc - this is the usual way to close a pop-up and so what most users tend to do.  Now you hear the bar 1, etc announced.
3. Repeat step 1 and this time press an arrow key to confirm with speech it has indeed jumped to bar 12.
4. Press Ctrl F and type 15-18, press enter.  Notice again no speech.  This time you can't confirm with arrows as navigation will cancel the selection!

When the selection made is for 1 note the usual string of note/bar/beat, etc should be heard the same as if you arrowed on to that note (I believe this did work in 4.5 or earlier.  When a range selection is made with the Go To command the range selection should be announced the same as when it is made with keyboard.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.2 beta

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by bkunda** [comment]
@DmitryArefiev could you please regression check this one?

**Comment by jmwmusic** [comment]
This is still a bug in 4.7.4 and was not fixed before release.
